desktop/caliban.py

```diff
@@ -3109,10 +3104,17 @@ def __init__(self, filename, raw, annotated, save_vars_mode):
         try:
             first_key = list(self.cell_info[0])[0]
             display_info_types = self.cell_info[0][first_key]
-            self.display_info = [*sorted(set(display_info_types) - {'frames'})]
+            if self.single_frame:
+                self.display_info = list((set(display_info_types) - {'frames'} - {'slices'}))
```

Comment:
I think this can be a single set.
```suggestion
                self.display_info = list((set(display_info_types) - {'frames', 'slices'}))
```